### ISSUE TITLE
Need to make Twelve of Code challenges

### ISSUE THREAD

**Issue by Mesure73L** [Mesure73L/mesure.x10.mx#60]
### Issue Contents

- [ ] March, Challenge 1
- [ ] March, Challenge 2
- [ ] March, Challenge 3

I will edit if we decide we should create the august challenges with the march ones.

**Comment by BeepBot99** [comment]
This is not high priority, there is a lot of time.
Getting everything to work is more high priority.

**Comment by BeepBot99** [comment]
@Mesure73L What is the theme for March? What about April?

**Comment by BeepBot99** [comment]
# Idea
Given: An Excel book
Find: Something based on the data in it

**Comment by BeepBot99** [comment]
@Mesure73L Maybe this?

**Comment by Mesure73L** [comment]
@BeepBot99 What is the march theme

**Comment by BeepBot99** [comment]
I don't know, what is it? Twelve of Code is your challenge, not mine.

**Comment by BeepBot99** [comment]
How about pick one of the four themes from #33.

**Comment by BeepBot99** [comment]
I added another theme to the discussion.

**Comment by BeepBot99** [comment]
I added two more themes to the discussion.

**Comment by BeepBot99** [comment]
@Mesure73L Does one of those sound good?

**Comment by Mesure73L** [comment]
Maybe the website one?

**Comment by BeepBot99** [comment]
The web crawler one?

**Comment by Mesure73L** [comment]
Yes

**Comment by Mesure73L** [comment]
Please make the program to make sure the websites are compatible with the maximum amount of loops

**Comment by BeepBot99** [comment]
What do you mean "the maximum amount of loops?"

**Comment by Mesure73L** [comment]
The maximum amount of times that the program needs to click on links for

**Comment by Mesure73L** [comment]
Make it a variable option

**Comment by BeepBot99** [comment]
Why would there be a maximum?

**Comment by BeepBot99** [comment]
And what do you mean a variable option?

**Comment by BeepBot99** [comment]
Or, how about instead of that, I write a program that finds the maximum for a given site?

**Comment by Mesure73L** [comment]
> Why would there be a maximum?

BECAUSE EVERYTHING RANDOM IS BASED OFF THE SEED

> And what do you mean a variable option?

an option in a variable 🤯

> Or, how about instead of that, I write a program that finds the maximum for a given site?

make it have an option in a variable (because you're too dumb to understand variable option) for both the website and the amount of times it should loop

**Comment by BeepBot99** [comment]
So you want me to write a program to solve the challenge?

**Comment by BeepBot99** [comment]
No, I'm saying, how about, given a website, it finds the maximum possible number **for that website** that works.

**Comment by Mesure73L** [comment]
> So you want me to write a program to solve the challenge?

yes, there will need to be one in EVERY SINGLE CHALLENGE to verify that the user was correct


> No, I'm saying, how about, given a website, it finds the maximum possible number **for that website** that works.

fine, as long as there's a way to verify that the website is compatible

**Comment by BeepBot99** [comment]
What do you mean compatible?

**Comment by Mesure73L** [comment]
TO CHECK IF IT WORKS WITH THE CHALLENGE AND WE CAN PUT IT IN THE LIST OF WEBSITES THAT THE USER CAN RECEIVE

**Comment by BeepBot99** [comment]
What makes a website work with the challenge?

**Comment by Mesure73L** [comment]
IF THE WEBSITE DOESN'T END BEFORE THE AMOUNT OF LOOPS THAT NEEDS TO BE COMPLETED

# 🤯🤯🤯🤯🤯

**Comment by Mesure73L** [comment]
Okay fine i'll give an example

lets say one website links to another website that has no links at all
how are you gonna continue if there's no more links to click?
thats why we need to make sure all the websites can reach the maximum
so that the users dont get confused

**Comment by BeepBot99** [comment]
So, what is the maximum then?

**Comment by BeepBot99** [comment]
I'm saying, we could have a JSON file with a list of all compatible websites, and the maximum for **each indivdual website**, but that is somehow too hard for you to understand. That way, it would be more random.

**Comment by BeepBot99** [comment]
I converted the third item in this issue to #92.

**Comment by BeepBot99** [comment]
Okay.

**Comment by BeepBot99** [comment]
Or what if, after the computer gets the output, the computer, based on that, gives the webpage another input. Then, the webpage has to return a specific HTTP status code based on that input.
The theme of March could be `Websites/HTTP`

**Comment by Mesure73L** [comment]
I don't understand.

**Comment by Mesure73L** [comment]
@BeepBot99 What should Mar 1 be?

**Comment by BeepBot99** [comment]
I don't know, whatever you want.

**Comment by Mesure73L** [comment]
What ideas do you have?

**Comment by Mesure73L** [comment]
> Well is March still math?

Yes, `Ma(rch|th)`.

**Comment by BeepBot99** [comment]
> [!IMPORTANT]
> Theme of march is `APIs`.

**Comment by Mesure73L** [comment]
> [!IMPORTANT]
> Theme of March is nonexistent. There is no march. Theme of April is `APIs`.
